dev/com.ibm.jbatch.container/.classpath

```diff
@@ -2,7 +2,7 @@
 <classpath>
 	<classpathentry kind="src" output="bin" path="src"/>
 	<classpathentry kind="src" output="bin_test" path="test"/>
-	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry exported="true" kind="con" path="aQute.bnd.classpath.container"/>
```

Comment:
Odd change.  Any reason?

Reply:
I don't remember making this myself, maybe came from doing something in Eclipse? I'll remove it and make sure it doesn't break anything.